### DISCUSSION TITLE
no illuminance sensor for TuYa ZG-205Z/A

### DISCUSSION THREAD

**Discussion by steinfahrer** [Koenkk/zigbee2mqtt#19518] [Devices]
Hell, I just got 2 items of the TuYa ZG-205Z. In the description at zigbee2mqtt there is a entity of "illuminance" described, but I cannot find any sensors for this item. 
Does it need to be implemented or does the TuYa ZG-205Z don't have that functionallity? At the package it is also discribed to have a "build in illuminance sensor".

**Comment by steinfahrer** [comment]
Here the discription from zigbee2mqtt:
https://www.zigbee2mqtt.io/devices/ZG-205Z_A.html

**Comment by elmaswebon** [comment]
Hi! Same problem here. Did you find any solution?

Thanks.
